Aptoide/appcoins-wallet-android
# [MOB-669] Backup incentives

**What does this PR do?**

Adds several improvements to the backup flow, and changes some icons and strings.

**Database changed?**

No

**Where should the reviewer start?**

BalanceFragment.kt

**How should this be manually tested?**

Just go through the Import and backup flow and check if everything is ok.
Give special attention to the tooltip and check for possible errors

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/MOB-669

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.)




**Code Review Checklist**
